qt/tools/genhooks_gui.py

```diff
@@ -392,6 +392,15 @@ def on_top_toolbar_did_init_links(links, toolbar):
         args=["player: aqt.sound.Player", "tag: anki.sound.AVTag"],
     ),
     Hook(name="av_player_did_end_playing", args=["player: aqt.sound.Player"]),
+    # Web
+    ###################
+    Hook(
+        name="will_init_bottom", 
```

Comment:
These would be clearer with the module at the start, eg

toolbar_bottom_will_init
toolbar_bottom_did_init